sdk/vision/azure-ai-vision-imageanalysis/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2024-09-06)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Add support for Entra ID authentication.
```

Comment:
We should also specify that we are now using `azure-json` dependency.

Reply:
Thanks! I added this under `Features added` section of CHANGELOG.md: 

* Migrate to stream-style serialization using the `azure-json` package. This is a new dependency introduced by `azure-core`